### ISSUE TITLE
Error No videos found play.tv3.lt/

### ISSUE THREAD

**Issue by filmaidykai** [spaam/svtplay-dl#1539]
### svtplay-dl versions:

490627:~$ svtplay-dl --version
svtplay-dl 4.20

### Operating system and Python version:

Name and version of the operating system and python version (run `python --version`)

Ubuntu 20.04.6 LTS

490627:~$ python --version
Python 2.7.18

Always include the URL you want to download and all switches you are using.
You should also add `--verbose` because it makes it much easier for use to find the issue :)

    490627:~$ svtplay-dl --verbose https://play.tv3.lt/series/elementaru-,serial-3837891/serija-1,episode-1032261
DEBUG [1685479163.3780727] /usr/lib/python3/dist-packages/svtplay_dl/utils/getmedia.py/get_media: version: 4.20
DEBUG [1685479163.379333] /usr/lib/python3/dist-packages/svtplay_dl/service/__init__.py/__init__: service: viafree
DEBUG [1685479163.3793879] /usr/lib/python3/dist-packages/svtplay_dl/utils/http.py/request: HTTP getting 'https://play.tv3.lt/series/elementaru-,serial-3837891/serija-1,episode-1032261'
DEBUG [1685479163.3911574] /usr/lib/python3/dist-packages/urllib3/connectionpool.py/_new_conn: Starting new HTTPS connection (1): play.tv3.lt:443
DEBUG [1685479163.5469816] /usr/lib/python3/dist-packages/urllib3/connectionpool.py/_make_request: https://play.tv3.lt:443 "GET /series/elementaru-,serial-3837891/serija-1,episode-1032261 HTTP/1.1" 200 None
DEBUG [1685479163.5798395] /usr/lib/python3/dist-packages/svtplay_dl/utils/http.py/request: HTTP getting 'https://viafree.mtg-api.com/identity/viafree/auth/pwd/sessions'
DEBUG [1685479163.5816321] /usr/lib/python3/dist-packages/urllib3/connectionpool.py/_new_conn: Starting new HTTPS connection (1): viafree.mtg-api.com:443
DEBUG [1685479163.7129657] /usr/lib/python3/dist-packages/urllib3/connectionpool.py/_make_request: https://viafree.mtg-api.com:443 "POST /identity/viafree/auth/pwd/sessions HTTP/1.1" 400 89
ERROR [1685479163.7137725] /usr/lib/python3/dist-packages/svtplay_dl/utils/getmedia.py/get_one_media: No videos found. You need to login

**Comment by filmaidykai** [comment]
svtplay-dl -S https://play.tv3.lt/series/stipresne-uz-lemti-,serial-5452989
ERROR: No videos found. You need to login

**Comment by rKsanu2MMYvypWePtQWM** [comment]
Can confirm, play.tv3.lt gives an error even when using credentials (python 3.11):

./svtplay-dl --verbose -u redacted -p redacted https://play.tv3.lt/local/demesio-centre-su-edmundu-jakilaiciu-,serial-4603730/2-sezonas-serija-1,episode-6045922
DEBUG [1694102498.5909214] /redacted/./svtplay-dl/svtplay_dl/utils/getmedia.py/get_media: version: 4.25
DEBUG [1694102498.5914426] /redacted/./svtplay-dl/svtplay_dl/service/__init__.py/__init__: service: viafree
DEBUG [1694102498.5914733] /redacted/./svtplay-dl/svtplay_dl/utils/http.py/request: HTTP getting 'https://play.tv3.lt/local/demesio-centre-su-edmundu-jakilaiciu-,serial-4603730/2-sezonas-serija-1,episode-6045922'
DEBUG [1694102498.592437] /usr/lib/python3.11/site-packages/urllib3/connectionpool.py/_new_conn: Starting new HTTPS connection (1): play.tv3.lt:443
DEBUG [1694102498.6193056] /usr/lib/python3.11/site-packages/urllib3/connectionpool.py/_make_request: https://play.tv3.lt:443 "GET /local/demesio-centre-su-edmundu-jakilaiciu-,serial-4603730/2-sezonas-serija-1,episode-6045922 HTTP/1.1" 200 None
DEBUG [1694102498.619813] /redacted/./svtplay-dl/svtplay_dl/utils/http.py/request: HTTP getting 'https://viafree.mtg-api.com/identity/viafree/auth/pwd/sessions'
DEBUG [1694102498.6205075] /usr/lib/python3.11/site-packages/urllib3/connectionpool.py/_new_conn: Starting new HTTPS connection (1): viafree.mtg-api.com:443
DEBUG [1694102499.8880825] /usr/lib/python3.11/site-packages/urllib3/connectionpool.py/_make_request: https://viafree.mtg-api.com:443 "POST /identity/viafree/auth/pwd/sessions HTTP/1.1" 401 None
ERROR [1694102499.8884554] /redacted/./svtplay-dl/svtplay_dl/utils/getmedia.py/get_one_media: No videos found. You need to login

**Comment by rudolphos** [comment]
Same problem

**Comment by latviantvg** [comment]
The problem still hasn't been fixed yet, it still doesn't work for me as well. Is there any possibility of a fix?

**Comment by posktomten** [comment]
Have you tried 
--username USERNAME
--password PASSWORD

**Comment by latviantvg** [comment]
I indeed have tried that (just like one of the people here earlier), but no result. It is the same, regardless of presence of the username and password.

**Comment by posktomten** [comment]
Prerequisites "Python 3.8 or higher"
https://svtplay-dl.se/install/source/
Perhaps?

**Comment by latviantvg** [comment]
No, this is most probably not related to that. All the needed things to get the video downloading are out-of-date, links now have play.tv3.lv/.lt/.ee not tv3play.lv/.lt/.ee as was mentioned by rudolphos earlier.
